rubicon/objc/core_foundation.py

```diff
@@ -209,6 +216,18 @@ def from_value(value):
         return at(value.decode('utf-8'))
     elif isinstance(value, Decimal):
         return NSDecimalNumber.from_decimal(value)
+    elif isinstance(value, dict):
+        dikt = NSMutableDictionary.alloc().init()
+        if value is not None:
```

Comment:
This is always true, because of the `isinstance` check before.

Reply:
Good point.